core/types/transaction.go

```diff
@@ -458,6 +458,21 @@ func NewTransactionsByPriceAndNonce(signer Signer, txs map[common.Address]Transa
 	}
 }
 
+// Copy copy a new TransactionsPriceAndNonce with the same *transaction
+func (t *TransactionsByPriceAndNonce) Copy() *TransactionsByPriceAndNonce {
+	heads := make([]*Transaction, len(t.heads))
+	copy(heads, t.heads)
+	txs := make(map[common.Address]Transactions, len(t.txs))
+	for acc, txsTmp := range t.txs {
```

Comment:
naming can be more clear.
`acc` stands for accountAddress?
`txs` -> `txsMap`?
`txsTmp` -> `txs`?

Reply:
as for `acc`, `txs`, just follow the naming convention say like in method `NewTransactionsByPriceAndNonce`